src/update-promoted-values.ts

```diff
@@ -124,16 +202,37 @@ function findPromotes(
       if (!yaml.isScalar(targetNode)) {
         throw Error(`Could not promote to ${[myName, ...collectionPath]}`);
       }
+
       const scalarToken = targetNode.srcToken;
       if (!yaml.CST.isScalar(scalarToken)) {
         // this probably can't happen, but let's make the types happy
         throw Error(
           `${[myName, ...collectionPath]} value must come from a scalar token`,
         );
       }
+
+      let relevantCommits: RelevantCommit[] = [];
```

Comment:
The block we're in right now runs for anything you're trying to promote between blocks — the default (and probably the only thing really used) is dockerImage.tag and gitConfig.ref. It's probably fine to hardcode this to *only* work for those paths, but we should recognize that the two cases need to be handled separately — the code you've written so far is just for dockerImage.tag, but we do also want (perhaps as a follow-up rather than the first thing we build) to get a list of commits that affect the Helm chart too.  (I think that's what you've done in github.ts that isn't called yet?)